lib/menu.c

```diff
@@ -828,6 +828,25 @@ bm_menu_run_with_key(struct bm_menu *menu, enum bm_key key, uint32_t unicode)
             }
             break;
 
+        case BM_KEY_PASTE:
+            {
+                FILE *clipboard;
+                clipboard = popen("wl-paste -t text/plain", "r");
+                if (clipboard == NULL) {
```

Comment:
minor thing, but the codebase uses `!cliboard` instead of `cliboard == NULL` style